src/test/java/de/tum/in/www1/artemis/lecture/AttachmentUnitsIntegrationTest.java

```diff
@@ -140,23 +128,17 @@ void splitLectureFile_asInstructor_createAttachmentUnits_shouldThrowError() thro
     @Test
     @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
     void splitLectureFile_asInstructor_shouldCreateAttachmentUnits_and_removeBreakSlides() throws Exception {
-        var filePart = createLectureFile(true);
-
-        LectureUnitInformationDTO lectureUnitSplitInfo = request.postWithMultipartFile("/api/lectures/" + lecture1.getId() + "/process-units", null, "process-units", filePart,
-                LectureUnitInformationDTO.class, HttpStatus.OK);
+        LectureUnitInformationDTO lectureUnitSplitInfo = request.postWithMultipartFile(lecture1.getId(), createLectureFile(true), OK);
         assertThat(lectureUnitSplitInfo.units()).hasSize(2);
         assertThat(lectureUnitSplitInfo.numberOfPages()).isEqualTo(20);
         lectureUnitSplitInfo = new LectureUnitInformationDTO(lectureUnitSplitInfo.units(), lectureUnitSplitInfo.numberOfPages(), true);
 
-        List<AttachmentUnit> attachmentUnits = List.of(request.postWithMultipartFile("/api/lectures/" + lecture1.getId() + "/attachment-units/split", lectureUnitSplitInfo,
-                "lectureUnitInformationDTO", filePart, AttachmentUnit[].class, HttpStatus.OK));
+        List<AttachmentUnit> attachmentUnits = request.postWithMultipartFile(lectureUnitSplitInfo, createLectureFile(true), lecture1.getId(), OK);
```

Comment:
here as well, we want to have tests be as fast as possible and to avoid unneded requests

Reply:
the second request takes the result of the first one as a parameter, -> cannot be omitted